### DISCUSSION TITLE
Volume in Linux and Windows Worker Nodes

### DISCUSSION THREAD

**Discussion by Premfer** [longhorn/longhorn#3703] [Q&A]
Hi,

we are using longhorn as a storage class in our cluster. Till now we had only linux nodes, now we are trying to add a windows node to run windows container. but we need a common volume to be mounted to windows node also. Is this possible ? will longhorn support this

**Comment by jenting** [comment]
Longhorn hasn't supported the Windows node because Longhorn relies on Linux operations.
BTW,  we have a [document](https://longhorn.io/docs/1.2.3/advanced-resources/deploy/rancher_windows_cluster/) to set up mixed cluster (Windows+Linux).
